shared/utility/math/geometry/Quad.h

```diff
@@ -43,7 +240,16 @@ namespace math {
              * @param right    The right x pixel value.
              * @param bottom   The bottom y pixel value.
              */
-            void set(double left, double top, double right, double bottom);
+            void set(const Scalar& left, const Scalar& top, const Scalar& right, const Scalar& bottom) {
```

Comment:
Should this be inline keyworded? If not, how come this implementation is on the header file /*legit question, because I thought that's against standard practice */ ?

Reply:
[https://en.cppreference.com/w/cpp/language/inline](https://en.cppreference.com/w/cpp/language/inline)

> A function defined entirely inside a class/struct/union definition, whether it's a member function or a non-member friend function, is implicitly an inline function.

Reply:
Ahh, it's implicit. Thanks!

Reply:
That page isn't actually correct. I'll fix it, it's implicit because it's a template not because it's a class